src/operations/property-filter.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+import { PropertyFilterProperty } from '..';
```

Comment:
Is it possible to import it from `./interfaces` to avoid circular imports?

Reply:
Sure, will fix